packages/app-store/_utils/downloadAsCsv.ts

```diff
@@ -0,0 +1,18 @@
+export const downloadAsCsv = (csvRaw: string, filename: string) => {
```

Comment:
We have this code elsewhere, can you re-use that or move this code to a re-usable function outside of app-store utils?

Reply:
addressed!